code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -143,28 +143,25 @@ Contains most of the procs that are called when a mob is attacked by something
 
 //Returns 1 if the attack hit, 0 if it missed.
 /mob/living/carbon/human/attacked_by(obj/item/I, mob/living/user, def_zone)
-	var/target_zone = def_zone? check_zone(def_zone) : get_zone_with_miss_chance(user.zone_selected, src)
+	var/target_zone
 
 	if(user == src) // Attacking yourself can't miss
 		target_zone = user.zone_selected
-	else if(!prob(user.melee_accuracy))
-		user.do_attack_animation(src)
-		playsound(loc, 'sound/weapons/punchmiss.ogg', 25, TRUE)
-		user.visible_message("<span class='danger'>[user] misses [src] with \the [I]!</span>", null, null, 5)
-		return FALSE
+	else
+		target_zone = def_zone? check_zone(def_zone) : get_zone_with_miss_chance(user.zone_selected, src)
 
 	var/datum/limb/affecting = get_limb(target_zone)
-	if(!affecting)
-		CRASH("[src] called attacked_by [user] with [I] aiming at [def_zone] but found no affecting limb")
 	if(affecting.limb_status & LIMB_DESTROYED)
 		to_chat(user, "What [affecting.display_name]?")
+		log_combat(user, src, "attacked", I, "(FAILED: target limb missing) (INTENT: [uppertext(user.a_intent)]) (DAMTYE: [uppertext(I.damtype)])")
```

Comment:
why should this be logged

Reply:
Same reason as the other times the attack failed due to missing?